frontend/src/main/scala/akkaviz/frontend/FrontendApp.scala

```diff
@@ -203,30 +205,27 @@ object FrontendApp extends JSApp with Persistence with PrettyJson with Manipulat
 
     repo.newActors.foreach {
       newActors =>
-        hierarchyView.insert(newActors)
+        hierarchyPanel.insert(newActors)
     }
 
+    setupComponents()
+  }
+
+  def setupComponents(): Unit = {
+    topMenu.attach(document.getElementById("left-panel"))
+    bottomMenu.attach(document.getElementById("left-panel"))
+    rightMenu.attach(document.getElementById("right-pane"))
     connectionAlert.attach(document.body)
-    actorSelector.attach(document.getElementById("actorselection"))
-    messageFilter.attach(document.getElementById("messagefiltering"))
-    messagesPanel.attach(document.getElementById("messagelist"))
-    asksPanel.attach(document.getElementById("asklist"))
-    document.getElementById("receivedelay").appendChild(receiveDelayPanel.render) //FIXME: port to component
-    monitoringOnOff.attach(document.getElementById("onoffsettings"))
-    unconnectedOnOff.attach(document.getElementById("graphsettings"))
-    replTerminal.attach(document.getElementById("repl-tab"))
-    graphView.attach(document.getElementById("graphview"))
-    hierarchyView.attach(document.getElementById("hierarchy-view"))
+    document.getElementById("globalsettings").appendChild(receiveDelayPanel.render) //FIXME: port to component
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Take the required action to fix the issue indicated by this comment.](https://www.codacy.com/app/lustefaniak/akka-viz/file/2349776081/issues/source?bid=3121282&fileBranchId=3217398#l219)